### ISSUE TITLE
TypeError: Cannot read property 'supressNlapiLogOutput' of undefined

### ISSUE THREAD

**Issue by sazr** [ProductEarthSolutions/netsumo#1]
*FYI this looks like a quite promising NetSuite testing module :) Hope it keeps getting maintained as I'm sure to share this with other NS developers.*

I'm following the NetSumo 'Getting Started' and trying to run the example suitelet test in my project. However I get an error when running the test:

> C:\Users\MyProject>mocha
> 
>   suitelet.js
>     #makeNetSuiteMagic
>       1) Adds a line item called My Magic Item and sets the quantity to 20
> 
>   0 passing (31ms)
>   1 failing
> 
>   1) suitelet.js #makeNetSuiteMagic Adds a line item called My Magic Item and sets the quantity to 20:
>      TypeError: Cannot read property 'supressNlapiLogOutput' of undefined
>       at nlapiLogExecution (node_modules\netsumo\modules\nsContext.js:20:13)
>       at Object.makeNetSuiteMagic (fileCabinet/suitelet.js:3:4)
>       at Context.<anonymous> (test\suiteletTest.js:20:17)

I editted `node_modules\netsumo\modules\nsContext.js` and simply commented out the possibly undefined option `supressNlapiLogOutput` and it now works.

```
  var nlapiLogExecution = function(type,title,details) {
    //if(!opts.supressNlapiLogOutput) {
      console.log("TYPE: "+type+" | TITLE: "+title+" | DETAILS: "+details)
    //}
  }
```

**Comment by matthewwilson** [comment]
Thanks for raising this @sazr 

The issue was caused by us not handling an undefined options object passed to the `nsContext.getDefaultContext(opts);` method.

I've added a check to handle this and published on NPM to version 1.4.6. I have also fixed the typo and made it `suppressNlapiLogOutput`!

FYI the options allow you to configure certain parts of the test framework for example:

```
var opts = {
  'suppressNlapiLogOutput': true
}

var context = nsContext.getDefaultContext(opts);
```

Will suppress log outputs to the terminal for the nlapiLogExecution method.

I will add more info to the readme.
